clients/src/test/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinatorTest.java

```diff
@@ -693,7 +693,6 @@ public boolean matches(ClientRequest request) {
 
         AtomicBoolean success = new AtomicBoolean(false);
         coordinator.commitOffsetsAsync(Collections.singletonMap(tp, new OffsetAndMetadata(100L)), callback(success));
-        consumerClient.poll(0);
```

Comment:
There are 3 more similar `consumerClient.poll(0)` calls. Any reason not to remove those as well?